package velrondevs.botania.common.lib;

import net.minecraft.client.resources.model.ModelResourceLocation;
import net.minecraft.resources.ResourceLocation;

public class ResourceLocationHelper {
	public static ResourceLocation prefix(String path) {
		return ResourceLocation.fromNamespaceAndPath(LibMisc.MOD_ID, path);
	}

	public static ModelResourceLocation modelResourceLocation(String path, String variant) {
		return new ModelResourceLocation(prefix(path), variant);
	}
}
